QuickGuidePanels/QuickGuidePanels.js

```diff
@@ -27,6 +27,15 @@ import WizardPanels from '../WizardPanels';
 const QuickGuidePanels = (props) => <WizardPanels fullScreenContent {...props} />;
 
 QuickGuidePanels.propTypes = {
+	/**
+	 * Hint string read when focusing the close button.
+	 *
+	 * @type {String}
+	 * @default 'Exit quick guide'
```

Comment:
Don't we need to wrap by $L?

Reply:
https://github.com/enactjs/sandstone/pull/1464/files#diff-4313679dd9c05a6dd7acdfaf794a2b0f0184323b7c8b384a3fed24533f2ed13dR479

There is $L already. For docs, some components describe $L also, but some components don't.

Reply:
Yes, that is my question. Shall we use &L on docs or not :)

Reply:
🤖 For now, the same manner as other "panels". In general, we need to talk.